### PR TITLE
subset of outcome metrics for `speculative-connect-sockets-increased-population`

### PR DESCRIPTION
**Pull request by mikewilli**
Workaround because custom experiment configs do not currently support overriding outcomes. Instead, we list out all of the metrics from the outcomes on the experiment, and set the `select_expression` to a no-op for metrics we don't want to compute.

### CODE REVIEW

**Review by scholtzan — APPROVED** [review]
Probably worth testing on a single day before we rerun everything just to realize that this doesn't work 😬

**Comment by mikewilli** [comment]
One thought I had while creating this -- wouldn't just not including the metrics in any of `[overall, weekly, daily]` exclude them from analysis? Or does the outcome definition include that and overrides what we have here?

**Comment by mikewilli** [comment]
This is blocked on https://github.com/mozilla/metric-config-parser/pull/335, so waiting for that and a subsequent Jetstream PR before merging.

**Comment by mikewilli** [comment]
This is good to merge now with https://github.com/mozilla/jetstream/pull/1988

**Pull request merged**
